python/GafferUI/RampPlugValueWidget.py

```diff
@@ -49,6 +49,9 @@ def __init__( self, plug, **kw ) :
 		GafferUI.PlugValueWidget.__init__( self, column, plug, **kw )
 
 		with column :
+			with GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 ) :
+				self.__basisLabel = GafferUI.LabelPlugValueWidget( plug["basis"] )
+				self.__basisField = GafferUI.SplineBasisWidget( plug["basis"], plug["endPointMultiplicity"])
 
```

Comment:
Could you try adding a spacer here, so that the menu doesn't dominate the layout quite so much? I think `GafferUI.Spacer( IECore.V2i( 0 ), parenting = { "expand" : True } )` might do it...